CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.0.3 (2022-01-24)
+
+- Fixed the angular tree component to not select the virtual root when clicking the select all button. [#56](https://github.com/blackbaud/skyux-angular-tree-component/pull/56)
```

Comment:
```suggestion
- Fixed the angular tree component to exclude the virtual root when users click the select all button. [#56](https://github.com/blackbaud/skyux-angular-tree-component/pull/56)
```
Minor wording tweaks. I think this reads more naturally, but it's not really a big deal. I'll go ahead and approve the PR in case you want to get it out without accepting these changes. ... If you make the changes and need another approval, please just let me know.